src/main/java/io/openshift/booster/catalog/BoosterCatalogService.java

```diff
@@ -344,16 +349,7 @@ public Path copy(Booster booster, Path projectRoot) throws IOException
 
    public List<Booster> getBoosters()
    {
-      Lock readLock = reentrantLock.readLock();
-      try
-      {
-         readLock.lock();
-         return boosters;
-      }
-      finally
-      {
-         readLock.unlock();
-      }
+      return new ArrayList<Booster>(boosters);
```

Comment:
No, because `boosters` is dynamic now. We can make the method return a `Collection` though and return `Collections.unmodifiableCollection(boosters)` (and _that_ we could cache).